client/my-sites/plugins-wpcom/purchase-button.jsx

```diff
@@ -15,7 +21,7 @@ export const PurchaseButton = React.createClass( {
 			);
 		}
 
-		return <Button compact primary>{ this.translate( 'Purchase' ) }</Button>;
+		return <Button compact primary onClick={ this.showPlansPage }>{ this.translate( 'Purchase' ) }</Button>;
```

Comment:
it looks like the `Button` component already accepts an `href` prop, which turns it into an `<a />` tag underneath the hood. Instead of tying this into `page()`, maybe we could just pass the URL instead.